sass/font-hack-subset.scss

```diff
@@ -1,3 +1,8 @@
+html {
+  /* Base font size */
+  font-size: 16px;
```

Comment:
Added an explicit base font size since everything else uses `rem`, it was already implicitly `16px`.